assets/i18n/qqq.json

```diff
@@ -80,7 +80,9 @@
 	"I18N_CREATE_NO_THANKS": "Text of the cancel button of a dialog. - see I18N_CREATE_EXPLORATION_CREATE.\n{{Identical|No thanks}}",
 	"I18N_CREATE_YES_PLEASE": "Text of the confirmation button of a dialog. - see I18N_CREATE_EXPLORATION_CREATE.",
 	"I18N_DASHBOARD_COLLECTIONS": "Title shown in a tab header in the user's dashboard page. - Clicking on this tab header shows a list of the collections the user has created.\n{{Identical|Collection}}",
+	"I18N_DASHBOARD_CREATOR_DASHBOARD": "Header that displays the text 'Creator Dashboard'",
```

Comment:
Please could you follow the existing format for qqq.json? Note that there are two parts: a description of the message, and some context about it.

Ditto below. Thanks!